### PR TITLE
add new apple squeeze method

### LINKED ISSUE #1: Double tap without a TextField
Hi!
Many thanks for this plugin. I have just tested it and realized that it only works for a text field. 

Do you think it is possible that it could also work without a text field?

### PR DESCRIPTION
**Pull request by AlexanderThiele**
closes #1

### CODE REVIEW

File: lib/entities/preffered_action.dart
Comment (NomiAdam) [inline lib/entities/preffered_action.dart:1]: **nit**, just a typo in file name `preffered_action.dart` > `preferred_action.dart`

**Comment by NomiAdam** [comment]
Other than the little nit it looks good to me 🚀

**Comment by AlexanderThiele** [comment]
thanks @NomiAdam 
fixed the typo.

but you might want to change the v1 callback method and align it maybe with the newer versions. i could not test it since my ipad was >= 17.5 already.

**Comment by AlexanderThiele** [comment]
@NomiAdam ping

**Comment by NomiAdam** [comment]
@AlexanderThiele Thank you for your work, appreciate it. I have been a bit busy so it didn't get to it yet, put it onto my todo list for this week. I will test everything out on my side to see if there is anything wrong and needs updating or not. I will update those issues accordingly 🤞

**Comment by NomiAdam** [comment]
Looks good 🚀

**Review by NomiAdam — APPROVED** [review]

**Pull request merged**